costmap_cspace_rviz_plugins/package.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>costmap_cspace_rviz_plugins</name>
+  <version>0.3.1</version>
+  <description>Rviz plugins for costmap_cspace_msgs</description>
+  <maintainer email="atsushi.w@ieee.org">Atsushi Watanabe</maintainer>
+  <license>BSD</license>
+  <license>Creative Commons</license>
```

Comment:
Creative Commons license in trajectory_tracker_rviz_plugins is for the icon image at https://github.com/at-wat/neonavigation_rviz_plugins/tree/master/trajectory_tracker_rviz_plugins/icons/classes
As this plugin doesn't use icon, it's not needed.

Reply:
Removed